VOM_Fortran/VOM-code/watbal.f90

```diff
@@ -189,7 +189,7 @@ subroutine waterbalance_fluxes ()
 
 !     * soil evaporation
 
-      esoil__  = 0.0002d0 * (1.d0 - 0.8d0 * (o_pct + pcg_d(2)))        &
+      esoil__  = 0.0002d0 * (1.d0 - 0.8d0 * (o_cai + pcg_d(2)))        &
```

Comment:
Need to look into the original motivation for this formulation and then possibly replace o_cai+pcg_d by the fraction of absorbed radiation by both trees and grasses.